src/Card/Card.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Link } from 'react-router-dom'
+
+import './Card.scss'
+
+interface CardProps {
+  name: string
+}
+
+const Card: React.FC<CardProps> = (props) => {
```

Comment:
I like that we have seen multiple ways to do this so far! I will mimic this going forward but happy to explore the initial way you completed this as well as a refactor if you'd like.